recipes-devtools/sam-ba/sam-ba_3.3.1.bb

```diff
@@ -4,7 +4,7 @@ LICENSE = "GPL-2.0"
 
 LIC_FILES_CHKSUM = "file://LICENSE.txt;md5=b234ee4d69f5fce4486a80fdaf4a4263"
 
-SRC_URI = "https://github.com/atmelcorp/${BPN}/releases/download/v${PV}/${BPN}_${PV}-linux_x86_64.tar.gz"
+SRC_URI = "https://github.com/atmelcorp/${BPN}/releases/download/v${PV}/${BPN}_${PV}-linux:x86_64.tar.gz"
```

Comment:
hi @kraj  . I don't think this is correct, did you use a script to replace '_' with ':'?

Reply:
ah that was an oversight fixed it now and yes there is a conversion script in oe-core scripts/contrib/convert-overrides.py which helps but the results need to be reviewed.